manifests/j/JGraph/Draw/21.7.5/JGraph.Draw.installer.yaml

```
@@ -1,21 +1,22 @@
-# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.LF.7-3-6.Unix
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
-
-PackageIdentifier: JGraph.Draw
-PackageVersion: 21.7.5
-InstallerLocale: en-US
-InstallerType: nullsoft
-Scope: machine
-InstallModes:
-- interactive
-- silent
-UpgradeBehavior: install
-Installers:
-- Architecture: x86
-  InstallerUrl: https://github.com/jgraph/drawio-desktop/releases/download/v21.7.5/draw.io-ia32-21.7.5-windows-32bit-installer.exe
-  InstallerSha256: D721AB9B15EBA605A94D0957453DF832BD174200995357C3E86A305ABE0B03E4
-- Architecture: x64
-  InstallerUrl: https://github.com/jgraph/drawio-desktop/releases/download/v21.7.5/draw.io-21.7.5-windows-installer.exe
-  InstallerSha256: 573158F7A5AC6FC1154E01C651D77CF1BD173B7BB1F05DFC25EE6D35DE7F9D22
-ManifestType: installer
-ManifestVersion: 1.5.0
+# Created with WinGet Automation using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: JGraph.Draw
+PackageVersion: 21.7.5
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-09-14
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/jgraph/drawio-desktop/releases/download/v21.7.5/draw.io-ia32-21.7.5-windows-32bit-installer.exe
+  InstallerSha256: D721AB9B15EBA605A94D0957453DF832BD174200995357C3E86A305ABE0B03E4
+- Architecture: x86
+  InstallerUrl: https://github.com/jgraph/drawio-desktop/releases/download/v21.7.5/draw.io-21.7.5-windows-installer.exe
+  InstallerSha256: 573158F7A5AC6FC1154E01C651D77CF1BD173B7BB1F05DFC25EE6D35DE7F9D22

```

This is the x64 version.
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/jgraph/drawio-desktop/releases/download/v21.7.5/draw.io-21.7.5-windows-installer.exe
  InstallerSha256: 573158F7A5AC6FC1154E01C651D77CF1BD173B7BB1F05DFC25EE6D35DE7F9D22
```